linux/buildScript.sh

```diff
@@ -11,12 +11,12 @@ set -u
 
 srctgz="https://www.census.gov/ts/x13as/unix/x13ashtmlsrc_V1.1_B19.tar.gz"
 file=$(basename ${srctgz})
-
 td=$(tempfile -p x13)
+rm -r $td
 mkdir ${td}
 cd ${td}
 wget ${srctgz}
-tar xfz ${file}
+tar xf ${file}
```

Comment:
As if changing file formats for a release weren't enough, putting a bogus file extension is really a cruel thing to do. Most major tar utilities (GNU, BSD, and busybox) will automatically detect the compression from the magic number, so dropping z should solve this as well as we can do without reading the minds of the US Census.